x-pack/platform/plugins/shared/ml/public/application/timeseriesexplorer/styles.ts

```diff
@@ -119,23 +119,23 @@ export const useTimeseriesExplorerStyles = () => {
             stroke: euiTheme.colors.mediumShade,
 
             '&.critical': {
-              fill: mlColors.critical,
+              fill: ML_SEVERITY_COLORS.CRITICAL,
             },
 
             '&.major': {
-              fill: mlColors.major,
+              fill: ML_SEVERITY_COLORS.MAJOR,
             },
 
             '&.minor': {
-              fill: mlColors.minor,
+              fill: ML_SEVERITY_COLORS.MINOR,
             },
 
             '&.warning': {
-              fill: mlColors.warning,
+              fill: ML_SEVERITY_COLORS.WARNING,
             },
 
             '&.low': {
-              fill: mlColors.lowWarning,
+              fill: ML_SEVERITY_COLORS.LOW,
```

Comment:
Just noting, it might be intentional as well but `ML_SEVERITY_COLORS.LOW` is different than `mlColors.lowWarning`

Reply:
Same as the other, comment above, not sure if this was intentional or oversight way back.